var/spack/repos/builtin/packages/jq/package.py

```diff
@@ -34,3 +34,4 @@ class Jq(AutotoolsPackage):
     version('1.5', 'c8070bd6ec275404f77db3d2e568c9a3')
 
     depends_on('oniguruma')
+    depends_on('bison@3.0:')
```

Comment:
If you want to tweak it, you can comment out config scopes in `lib/spack/spack/config.py:205`.  I feel like there was a PR to add a config scope specifier on the command line.  Maybe that was to add an additional scope, though, not to ignore.